src/main/java/com/Studentlifr/GratitudeBoard/services/AdminServices.java

```diff
@@ -0,0 +1,19 @@
+package com.Studentlifr.GratitudeBoard.services;
+
+import com.Studentlifr.GratitudeBoard.dao.UserDAO;
+import com.Studentlifr.GratitudeBoard.dto.response.AdminMessagesResDTO;
+import com.Studentlifr.GratitudeBoard.dto.response.GenericResDTO;
+import com.Studentlifr.GratitudeBoard.dto.response.ResponseDTO;
+
+import java.util.List;
+import java.util.UUID;
+
+public interface AdminServices {
+    public GenericResDTO<List<AdminMessagesResDTO>> getAllMessages();
```

Comment:
remove public here as any method declared here is public by default

Reply:
Removing Public from there, restricting me to access the admin service package in controller, (Outside the Package).